Source/SvgElement.cs

```diff
@@ -968,29 +968,29 @@ ISvgNode ISvgNode.DeepCopy()
 
         public virtual SvgElement DeepCopy<T>() where T : SvgElement, new()
         {
-            var newObj = new T();
-            newObj.ID = this.ID;

```

Comment:
I guess the ID does not need to be copied?

Reply:
`ID` is an attribute, so it is copied.

Reply:
Ah yes, thanks!